File: Base Dati/libreria/libro.py

class Libro:

    def __init__(self, id, codice, autore, titolo, editore, classificazione) -> None:
        self.id = id
        self.codice = codice
        self.autore = autore
        self.titolo = titolo
        self.editore = editore
        self.classificazione = classificazione

    def __str__(self) -> str:
        return f"""
        Titolo : {self.titolo}
        Autore : {self.autore}
        Classificazione : {self.classificazione}
        """

File: php/L04/java/Libreria/src/Libro.java

public class Libro {

    public String titolo;
    public String autore;
    public int annoPubblicazione;

    public Libro(String titolo, String autore, int annoPubblicazione) {
        this.titolo = titolo;
        this.autore = autore;
        this.annoPubblicazione = annoPubblicazione;
    }

    @Override
    public String toString() {
        return "Titolo: " + this.titolo + ", Autore: " + this.autore + ", Anno di Pubblicazione: " + this.annoPubblicazione;
    }
}
